cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerCapacityStats.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.response.stats;
+
+import java.util.Map;
+
+
+public class SingleBrokerCapacityStats {
+  private static final String HOST = "Host";
+  private static final String BROKER = "Broker";
```

Comment:
we have the same macro in `SingleBrokerStats`, also there are some same method(`host()`,`id()`), we probably need a base class here to hold the common things here.